Packs/jamf/ParsingRules/jamf/jamf.xif

```diff
@@ -1,3 +1,3 @@
 [INGEST:vendor="jamf", product="pro", target_dataset="jamf_pro_raw", no_hit=keep]
-filter webhook ~= "\d{7,13}"
+filter webhook != null AND to_string(webhook) ~= "eventTimestamp[^\d]+\d{7,13}"
```

Comment:
maybe we could replace the _[^\d]_ with _\D_ or first try to extract the _eventTimestamp_ and apply the filter on that field directly - 
```c
| alter tmp_eventTimestamp = webhook -> eventTimestamp
| filter to_string(tmp_eventTimestamp) ~= "\d{10}|\d{13}|\d{16}|\d{19}" 
```

On any case I believe the epoch should be at least 10 digits long.

Reply:
Another form of implementation, I am using less alter operation, so maybe this offer is less effective..
Not critical.